include/aws/common/common.h

```diff
@@ -41,6 +41,7 @@
 #endif
 
 #define AWS_CONCAT(A, B) A ## B
+#define GHOST(x) ghost_ ## x
```

Comment:
Please put this in the CBMC conditional block in byte_buf.h, this will not be useful for anyone else.